src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -130,33 +135,31 @@ public static void setField(
   public static <T> Object getInstanceField(
       final Class<T> c,
       final Object o,
-      final String fieldName)
-      throws
-      NoSuchFieldException,
-      IllegalArgumentException,
-      IllegalAccessException {
+      final String fieldName) {
     if (c == null) {
-      throw new NoSuchFieldException();
+      throw new DeeptestUtilsException(fieldName + " is not a field");
```

Comment:
Any reason not to put the `new NoSuchFieldException()` into the cause?

Reply:
I second @thk123 's opinion here, would that also remove the need for `(String)` constructor of the custom exception